game/game.py

```diff
@@ -493,6 +496,40 @@ def position_culled(self, pos: Point) -> bool:
                 return False
         return True
 
+    def iads_considerate_culling(self, tgo: TheaterGroundObject[Any]) -> bool:
+        if not self.settings.perf_do_not_cull_threat_IADS:
+            return self.position_culled(tgo.position)
+        else:
+            if self.settings.perf_culling:
+                if isinstance(tgo, EwrGroundObject):
+                    max_detection_range = tgo.max_detection_range().meters
+                    for z in self.__culling_zones:
+                        seperation = z.distance_to_point(tgo.position)
+                        # EWR detection range is very far.  If left at 100% range no EWR would ever be culled.
```

Comment:
That's probably the right way to do it though? EWR sites are pretty simple groups with just one unit. I expect culling these doesn't buy as much as we break by removing the backbone of the IADS?

Reply:
The reason I did not leave it at 100% was I saw some campaigns had quite a few EWRs throughout.  Do you think it's inconsequential to performance even if there are a large number of EWRs?

Reply:
I don't know but without evidence to the contrary I'm hesitant to push for it. Skynet needs those EWRs to function well, and not all campaigns do that, so there are other options.

I'm probably not a very helpful reviewer for anything related to culling. I don't use it because I don't think it's worth the ways it breaks the game. Dedicated servers improve performance to the point where I don't see the need.

Reply:
> I'm probably not a very helpful reviewer for anything related to culling. I don't use it because I don't think it's worth the ways it breaks the game. Dedicated servers improve performance to the point where I don't see the need.

I've only tried it once, but I had worse performance running a dedicated server on the same computer as the DCS client.